src/pages/main/components/itemList/Item.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Item = ({ key, productNameEn, productNameKr, price, src }) => {
+  const getRandomNumber = max => {
+    return (Math.random() * max).toFixed(1);
+  };
+
+  return (
+    <ItemComponent>
+      <ItemImg src={src} />
+      <ItemTitle>
+        <Description>
+          <ItemNameEn>{productNameEn}</ItemNameEn>
+          <ItemNameKr>{productNameKr}</ItemNameKr>
+        </Description>
+        <Amount>
+          <Price>{price}원</Price>
+          <PriceDescription>즉시 구매가</PriceDescription>
+        </Amount>
+      </ItemTitle>
+      <Interests>
+        <BookMark>
+          <BookMarkIcon src="/images/ItemList/bookmark.png" alt="icon" />
+          <BookMarkNum>{getRandomNumber(20)}만</BookMarkNum>
+        </BookMark>
+        <Likes>
+          <LikesIcon src="/images/ItemList/like.png" alt="icon" />
+          <LikesNum>{getRandomNumber(10)}만</LikesNum>
+        </Likes>
+      </Interests>
+    </ItemComponent>
+  );
+};
+
+export default Item;
+
+const ItemComponent = styled.div`
+  margin-bottom: ${props => props.theme.margins.xxxl};
+`;
+
+const backgourndColors = [
+  'lavender',
+  'lavenderblush',
+  'linen',
+  'powderblue',
+  'thistle',
+];
+const getRandomColor = () => {
+  return backgourndColors[Math.floor(Math.random() * backgourndColors.length)];
+};
+
+const ItemImg = styled.img`
+  width: 14rem;
+  height: 14rem;
+  align-items: center;
+  justify-content: center;
+  background-color: ${getRandomColor()};
```

Comment:
함수를 실행한 결과값이 아니라, 매번 랜덤으로 실행하는 함수를 전달해야 할 것 같습니다.
```suggestion
  background-color: ${getRandomColor};
```

Reply:
수정했습니다!